app/adapters/application.js

```diff
@@ -1,3 +0,0 @@
-import LFAdapter from 'ember-localforage-adapter/adapters/localforage';
```

Comment:
It would be fine to add it in but I find it more clear use the addon namespace when including files.

For example.

```
//example-application/app/adapters/application.js
import Adapter from 'ember-localforage-adapter/adapters/localforage';
export default Adapter;
````

vs
````
//example-application/app/adapters/application.js
import Adapter from '/example-application/adapters/localforage';
export default Adapter;
````

The only file that needs to be injected into the application namespace is the serializer.  Only for this [line](https://github.com/genkgo/ember-localforage-adapter/blob/master/addon/adapters/localforage.js#L7);